app/app/static/custom/ourscript.js

```diff
@@ -114,6 +115,51 @@ const terminateSession = async(data) => {
 	}
 }
 
+function validateUserInput(data) {
+	// Check student name
+	if (data.studentName.length > 50) {
+		alert("'Student Name' field should is too long.");
+		return false;
+	}
+	for(let i = 0; i < data.studentName.length; i++) {
+		if (!isNaN(parseInt(data.studentName[i]))) {
+			alert("Please do not use numeric values in 'Student Name' field.");
+			return false;
+		}
+	}
```

Comment:
this is better taken care of with Regex https://stackoverflow.com/questions/4440286/numbers-not-allowed-0-9-regex-expression-in-javascript

Reply:
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test for regex test javascript